test/membrane/reversed-proxy-constructor.spec.js

```diff
@@ -0,0 +1,32 @@
+import createSecureEnvironment from "../../lib/browser-realm.js";
+
+describe("Reversed Proxy constructor", () => {
+    it("can be constructed", () => {
+        const evalScript = createSecureEnvironment({
+            endowments: {
+                test({ Proxy }) {
+                    const p = new Proxy({}, {
+                        get() { return 1 }
+                    });
+                    expect(p.a).toBe(1);
+                },
+            },
+        });
+        evalScript(`test({ Proxy });`);
+    });
+    it(".revocable() should be supported", () => {
+        const evalScript = createSecureEnvironment({
+            endowments: {
+                test({ Proxy }) {
+                    const { proxy, revoke } = Proxy.revocable({}, {
+                        get() { return 1 }
+                    });
+                    expect(proxy.a).toBe(1);
+                    revoke();
```

Comment:
☝️ This is weird. `proxy.a` should be `1` but is `undefined` not sure why.

Reply:
we will have to figure this out... weird indeed

Reply:
did you figure why? code is looking great and tests are passing.

Reply:
I have not yet. The jest tests are passing but the karma ones are not (CI just does jest tests)